packages/ember-principled-forms/addon/components/form-field/component.ts

```diff
@@ -87,25 +89,26 @@ export default class FormField<T> extends Component {
     return !this.isInvalid ? undefined : 'true';
   }
 
-  // DISCUSS: What other useful defaults can we apply?
-  // e.g. an ARIA object
-  id: string = defaultTo(this.id, `form-field-${this.model.type}-${this.label}`);
-  errorId: string = defaultTo(
-    `${this.id}-error`,
-    `form-field-${this.model.type}-${this.label}-error`
-  );
-
   @computed('isInvalid')
   get ariaDescribedBy(): HTMLAttribute {
     return this.isInvalid ? this.errorId : undefined;
   }
 
-  onChange: (newValue: T) => void = defaultTo(this.onChange, noop);
-  onInput: (newValue: T) => void = defaultTo(this.onInput, noop);
+  onChange!: (newValue: T) => void;
```

Comment:
It probably makes sense to move these up with the other declarations, possibly with comments for which are based on args and which aren't.